samples/boards/reel_board/mesh_badge/src/periphs.c

```diff
@@ -138,8 +138,8 @@ static void configure_accel(void)
 	}
 
 
-	k_delayed_work_init(&motion_work, motion_timeout);
-	k_delayed_work_submit(&motion_work, MOTION_TIMEOUT);
+	k_work_init_delayable(&motion_work, motion_timeout);
+	k_work_reschedule(&motion_work, MOTION_TIMEOUT);
```

Comment:
This could be `k_work_schedule()` since there is no earlier timeout that might need to be overridden.